daily_post.py

```diff
@@ -21,25 +21,35 @@
 
 now = datetime.datetime.now()
 
+
 def get_weather():
     resp = requests.get('http://forecast.weather.gov/MapClick.php?lat=47.62&lon=-122.36&unit=0&lg=english&FcstType=text&TextType=1')
     doc = resp.content
     soup_doc = BeautifulSoup(doc, 'html.parser')
-    weather_info = soup_doc.find_all('table')[1]
-    forecast = weather_info.contents[0].text
-    reddit_comment = '* ' + str.replace(forecast, '\n\n',"\n* ")[:-3]
+    weather_info = soup_doc.find_all('table')[1].contents[0].contents[0]
+    reddit_comment = ''
+    if len(weather_info.find_all('div')) > 0:
+        advisories = weather_info.div.extract().find_all('a')
+        reddit_comment += '* Advisories: \n'
```

Comment:
OCD pickiness on my front, no need for the space after the `:`

Reply:
Oh also, you can probably also just do `if weather_info.find_all('div')` since it would return null if nothing was found I believe.